### PR TITLE
Issue #1809 Jamendo scrobbler error fixed

### PR DESCRIPTION
**Pull request by NamJinWoo**
Issue #1809 Jamendo scrobbler error fixed

Issue https://github.com/web-scrobbler/web-scrobbler/issues/1809

Play button and TrackArt were not working.

So, I fixed it like this:
Connector.isPlaying = () => {
return $('.js-player-play-pause > .icon-pause').length !== 0;
};

Connector.getTrackArt = () => {
return ${$('.hero-cover').find('img').attr('src')};
};

I compiled it in my local pc, and it works now.

Thanks:)

### CODE REVIEW

**Comment by NamJinWoo** [comment]
I had a conflicts in radioparadise.js because I was trying to solve the problems that had been occured before. I rolled it back to the one before.

Jamendo site error is fixed now.

For the issue #1809 

Thanks:)

File: src/connectors/jamendo.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ Connector.currentTimeSelector = '.hidden-xs > .js-player-position';
 
 Connector.durationSelector = '.hidden-xs > .js-player-duration';
 
-Connector.playButtonSelector = '.js-player-play-pause > .icon-play';
+Connector.isPlaying = () => {
+	return $('.js-player-play-pause > .icon-pause').length !== 0;
+};
 
-Connector.trackArtSelector = '.js-full-player-cover-img';
```
Comment (inverse) [inline src/connectors/jamendo.js]: Wouldn't changing this selector to `.hero-cover img` be better?

Comment (NamJinWoo) [reply src/connectors/jamendo.js]: That also works. I tried to get a "src" from image. I guess both works.

What do you prefer more? If you prefer that one more, please tell me. I will take it over.

**Comment by NamJinWoo** [comment]
I tried what you said, and compiled it in my local laptop. And it works too.

I just changed it into that code, and sent you a PR again.

Please check! Thanks:)

**Comment by inverse** [comment]
Thanks!

**Pull request merged**